src/main/java/spark/Service.java

```diff
@@ -130,6 +131,16 @@ public synchronized Service port(int port) {
     }
 
     /**
+     * Set whether to enable the underlying server's JMX counters
+     * @param enableServerJMX
+     * @return
+     */
+    public synchronized Service enableServerJMX(boolean enableServerJMX) {
```

Comment:
Makes sense - will do. I don't have numbers comparing the two as the system in use previously had JMX stats enabled so we had to have them on in new code from the beginning to maintain monitoring parity.